commands/Moderation/nickname.js

```diff
@@ -0,0 +1,22 @@
+exports.run = async (client, msg, [user, name = ""]) => {
+  await user.setNickname(name);
+  return msg.reply(`Nickname has been changed to ${name} for ${user} 😄`);
```

Comment:
If the nickname is empty, it'll reset. That'll make it output this:

```javascript
Nickname has been changed to  for @Skill 😄
```

You might want to check if `name.length === 0`, if so, say nickname cleared. Otherwise what this command already says.

Reply:
Also, `msg.send`